core/domain/user_services.py

```diff
@@ -568,19 +575,30 @@ def __init__(self, user_id=None):
 
     @property
     def user_id(self):
+        """Returns:
+            user_id: str. A unique ID of user.
+        """
```

Comment:
Please write 'Returns the unique ID of the user' before 'Returns:' as it was previously. We should include both the description about the method and its 'Args:' and 'Returns:'.

Reply:
Done @vibhor98 :)